services/movie_session.py

```diff
@@ -0,0 +1,51 @@
+from django.utils.dateparse import parse_date
+from django.db.models import QuerySet
+from db.models import MovieSession
+from datetime import datetime
+
+
+def create_movie_session(
+    movie_show_time: datetime,
+    movie_id: int,
+    cinema_hall_id: int
+) -> MovieSession:
+    new_session = MovieSession(
+        show_time=movie_show_time,
+        movie_id=movie_id,
+        cinema_hall_id=cinema_hall_id
+    )
+    new_session.save()
+    return new_session
+
+
+def get_movies_sessions(session_date: str = None) -> QuerySet:

```

Comment:
If argument can be `None` you should show it in type hint
```suggestion
def get_movies_sessions(session_date: str | None = None) -> QuerySet[MovieSession]:

```